### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by tommoor**

### CODE REVIEW

File: shared/i18n/locales/fr_FR/translation.json [hunk 1]
```diff
@@ -433,12 +433,12 @@
   "Publishing": "Publication en cours",
   "No history yet": "Pas encore d'historique",
   "Stats": "Statistiques",
-  "{{ count }} minute read": "{{ count }} minute read",
-  "{{ count }} minute read_plural": "{{ count }} minute read",
+  "{{ count }} minute read": "Durée de lecture de {{ count }} minute",
+  "{{ count }} minute read_plural": "Durée de lecture de {{ count }} minutes",
```
Comment (bmm-alc) [inline shared/i18n/locales/fr_FR/translation.json:437]: there is a typo here, it should be "minutes"

Comment (tommoor) [reply shared/i18n/locales/fr_FR/translation.json:437]: Do you mean in the key name? Those are kept singular as part of the internationalization framework

Comment (bmm-alc) [reply shared/i18n/locales/fr_FR/translation.json:437]: Ok, but in the crowndin UI both strings (singular and plurial) appears as the same, and there is no hint the second was for the plural form.
image

Comment (tommoor) [reply shared/i18n/locales/fr_FR/translation.json:437]: Ah weird, I suppose it's because it's the same in english.

**Pull request merged**